microsoft-edge/progressive-web-apps-chromium/how-to/digital-goods-api.md

```diff
@@ -105,20 +143,36 @@ The item ID is a string that represents the primary key of the items.  In the Mi
 
 The item's `price` is a `PaymentCurrencyAmount` that contains the current price of the item in the user's current region and currency.  The `price` is designed to be formatted for the user's current locale by using `Intl.NumberFormat`, as shown above.
 
-See also:
+
+<!-- ------------------------------ -->
+#### See also
+
+Digital Goods API specification
+* [getDetails() method](https://wicg.github.io/digital-goods/#getDetails-method)
+
+Windows App Development:
 * [StoreProduct.InAppOfferToken Property](/uwp/api/windows.services.store.storeproduct.inappoffertoken)
-* [PaymentCurrencyAmount dictionary](https://www.w3.org/TR/payment-request/#dom-paymentcurrencyamount) in _Payment Request API_ at W3C.
-* [Intl.NumberFormat](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat) at MDN.
+
+W3C:
+* [PaymentCurrencyAmount dictionary](https://www.w3.org/TR/payment-request/#dom-paymentcurrencyamount)
+
+MDN:
+* [Intl.NumberFormat](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat)
 
 
 <!-- ====================================================================== -->
-## Purchasing an item (`show` method)
+## Purchasing an item (`PaymentRequest.show` method)
 
-Use the `show` method to purchase an item, after you construct a request that contains the item details.
+Use the `show` method of the `PaymentRequest` interface to purchase an item, after you construct a request that contains the item details.
 
 Once your products and details are displayed to the user, you can implement the purchase flow by using the Payment Request API.  When combined with the Digital Goods API, the only required input parameter is `methodData`.
 
-Use the `supportedMethods` member of the `methodData`⁠⁠ parameter in the `PaymentRequest` to identify Microsoft Store Billing as the payment method with the string `"https://store.microsoft.com/billing"`.  Then in the `data` member, pass along the item ID as the `sku`.
+In the `PaymentRequest` constructor's `methodData`⁠⁠ parameter (type =`PaymentRequestEvent`):
+* Use the `supportedMethods` member to identify Microsoft Store Billing as the payment method with the string `"https://store.microsoft.com/billing"`.
+* In the `data` member, pass along the item ID as the `sku`:
+
+<!-- todo: in PaymentRequest ctor, the 2nd param (details) is missing, per ctor docs: https://developer.mozilla.org/docs/Web/API/PaymentRequest/PaymentRequest - the 3rd param, "options", says "optional" next to it, unlike "details" param - is that page missing a 1-param overload?  new PaymentRequest(methodData)
```

Comment:
in the code listing, in the PaymentRequest ctor, the 2nd param (details) is missing, per ctor docs: https://developer.mozilla.org/docs/Web/API/PaymentRequest/PaymentRequest - the 3rd param, "options", says "optional" next to it, unlike "details" param - is that page missing a 1-param overload?:
new PaymentRequest(methodData)

@runyuany 

rendered section:
https://review.learn.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/how-to/digital-goods-api?branch=pr-en-us-3324#purchasing-an-item-paymentrequestshow-method

Reply:
above in this section:
"When combined with the Digital Goods API, the only required input parameter is methodData."